Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextToolbar/IsFormat.cs

```diff
@@ -0,0 +1,33 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarFormats;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages.TextToolbar
+{
+    public class IsFormat : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            return (Format)value == Format;
```

Comment:
This was a quick converter in the Sample App, but I could make it available for use with the toolbar.